dasharo/releases.yaml

```diff
@@ -1,4 +1,17 @@
 releases:
+        - version: v1.0.16
+          added:
+                - Vboot Verified + Measured Boot
+                - Vboot recovery notification in TianoCore
+                - UEFI Shell
+                - UEFI Secure Boot
+                - Intel PTT Firmware TPM support
+          changed:
+                - FSP changed to CometLake2
+          known_issues:
+                - description: i225 network controller initialization takes too much time
```

Comment:
We can easily fix it, see PROT-462 @macpijan

Reply:
Thanks for reminding

Reply:
Fixed, moved this issue to the `fixed` section